src/Display/Tabs/Tabs.test.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+
+import * as renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { fireEvent, render } from '@testing-library/react';
+
+import Tabs from './Tabs';
+import { EHorizontalTabVariant } from '../../Utils/StyleConfig';
+import { SecondaryColor } from '../../Style/Colors';
+
+describe('<Tabs/> render', () => {
+  test('should match snapshot', () => {
+    const snapshot = renderer
+      .create(
+        <Tabs>
+          <Tabs.Pane tab="Job">Software Engineer</Tabs.Pane>
+          <Tabs.Pane tab="Company">Glints</Tabs.Pane>
+          <Tabs.Pane tab="Location">Jakarta</Tabs.Pane>
+          <Tabs.Pane tab="Salary">Rp 10,000,000</Tabs.Pane>
+        </Tabs>
+      )
+      .toJSON();
+    expect(snapshot).toMatchSnapshot();
+  });
+});
+
+describe('<Tabs/> should render correctly with the alignment prop', () => {
+  test('horizontal, when alignment is set horizontal', () => {
+    const { getByRole } = render(
+      <Tabs alignment="horizontal">
```

Comment:
I suggest testing all the possible `alignment`s, you can check `Divider.test.tsx`, import `jest-styled-components`, and generate snapshots with css included, so we don't need to check individual style. Just remember to check the snapshot and validate the css is being correctly generated.

Reply:
This is so much better than testing appropriate styes individually. Thanks! Updated now.